websites/D/Discord Türkiye/presence.ts

```diff
@@ -0,0 +1,115 @@
+const presence = new Presence({ clientId: "890757020393816064" });
+
+presence.on("UpdateData", async () => {
+	const presenceData = {
+			largeImageKey: "1024dctr",
+			details: "Geziniyor",
+			state: "",
+			startTimestamp: Math.floor(Date.now() / 1000),
+			smallImageText: "",
+			smallImageKey: "",
+		},
+		page = document.location.pathname,
+		privacyMode = await presence.getSetting("privacy_mode").catch(() => {
+			false;
+		});
+
+	if (document.querySelector('[href="/login/"]') != null) {
+		presenceData.smallImageText = "Misafir Modu";
+		presenceData.smallImageKey = "https://i.hizliresim.com/385dt7x.png";
+	} else if (document.querySelector("span.avatar.avatar--xxs > img")) {
+		presenceData.smallImageText = document
+			.querySelector("span.avatar.avatar--xxs > img")
+			.getAttribute("alt");
+		presenceData.smallImageKey = "https://i.hizliresim.com/385dt7x.png";
+	}
+
+	if (page == "/") presenceData.details = "Anasayfayı görüntülüyor";
```

Comment:
don't use lazy comparisons 
```suggestion
	if (page === "/") presenceData.details = "Anasayfayı görüntülüyor";
```